native/effektio/src/api/conversation.rs

```diff
@@ -206,11 +168,14 @@ impl ConversationController {
         }
     }
 
-    pub fn load_rooms(&self, convos: &Vec<Conversation>) {
-        for convo in convos.iter() {
-            convo.load_latest_message();
+    pub async fn load_rooms(&mut self, convos: &Vec<Conversation>) {
+        let mut conversations: Vec<Conversation> = vec![];
+        for convo in convos {
+            let mut conversation = convo.clone();
+            conversation.fetch_latest_message().await;
```

Comment:
should we really delay the returning of the entire list just to have the latest message available on each? I'd rather we moved this off into a background task - but then it can't be `mut` anymore. so, for now, I am fine with this, but we might want to put down a ticket that we want to optimise this for first-view-speed.

Reply:
I implemented flutter fetch function that is called from conversation list receiver, but it occurred recursive event callback.
I couldn't find that reason.
So I removed flutter fetch function and processed fetching in rust `ConversationController::load_rooms()` function.
You can see it here: https://github.com/effektio/effektio/pull/321/commits/b1f83e2829af4c20c28daeb0c1fbe7d660e740bf

Reply:
I will remember and deal with this problem again later.